app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -35,7 +35,7 @@
       <% if SETTINGS[:organizations_enabled] -%>
         <td class="hidden-tablet hidden-xs"><%= host.organization.try(:name) %></td>
       <% end -%>
-      <td class="hidden-tablet hidden-xs"><%= host.subnet %></td>
+      <td class="hidden-tablet hidden-xs"><%= host.primary_interface.subnet %></td>
```

Comment:
We could do `host.primary_intarface.try(:subnet)` maybe or similar with a comment what this is for.